portal/src/texts/profile/colors.md

```diff
@@ -1,78 +1,93 @@
+**Støttefargene** er forbeholdt illustrasjoner og små detaljer (e.g. farger i en graf). De skal ikke benyttes på store flater. 
 
-**Støttefargene** brukes i hovedsak til illustrasjoner og figurer (slik som diagrammer i presentasjoner og rapporter), men kan også brukes sparsomt for å skape oppmerksomhet rundt viktig innhold.
+**Funksjonsfargene** brukes i digitale flater når det er behov for flere valører enn det som finnes i hoved- og støttefargene, eller for å tilfredsstille krav om tilgjengelighet (e.g. røde feilmeldinger mot lys bakgrunn).
 
-**Funksjonsfargene** brukes i digitale flater, der man trenger å vise funksjonalitet ved hjelp av farge, eller har behov for flere valører enn det som finnes i hoved- og støttefargene. Disse inkluderer blant annet rød-, grønn- og blåtoner som tilfredsstiller krav om tilgjengelighet.
 
 ### Farger i digitale løsninger
 
-I digitale løsninger benytter vi oss av våre hovedfarger og funksjonsfarger. Disse gjør det enkelt å sette opp et visuelt hierarki og sikre god kontrast. Støttefargene våre er i all hovedsak forbeholdt illustrasjoner og grafikk. De kan også sparsomt brukes til å skape litt mer liv og varme i løsningene våre, men spør alltid om det kan heller løses via våre hovedfarger.
+I digitale løsninger benytter vi oss av våre hovedfarger og funksjonsfarger. Støttefargene er i utgangspunkt forbeholdt illustrasjoner og grafikk men kan veldig sparsomt brukes til å fremheve informasjon i løsningene våre. Spør alltid om det kan heller løses via våre hovedfarger.
 
-#### Tilgjengelighet
 
-Tilgjengelighet er det viktigste å tenke på når du benytter fargene i digitale løsninger. Vi skal alltid etterkomme kravene til universell utforming, som for farger vil bety [AA-kravet for kontrast](link). Kontrastkravet gjelder ikke bare tekst på bakgrunn, men også komponentenes kontrast mot sin bakgrunn (e.g. ikke bare tekst på knappebakgrunnen, men også knappebakgrunnen mot sidens bakgrunnsfarge).
+### Tilgjengelighet
+
+Tilgjengelighet er det viktigste å tenke på når du benytter fargene i digitale løsninger. Vi skal alltid etterkomme kravene til universell utforming, som for farger vil bety [AA-kravet for kontrast](link). Kontrastkravet gjelder ikke bare tekst på bakgrunn, men også komponentenes kontrast mot sin bakgrunn når den er av betydning for å forstå funksjonaliteten (e.g. ikke bare tekst på knappebakgrunnen, men også knappebakgrunnen mot sidens bakgrunnsfarge).
 
 Tekstfargene våre er helhvit eller svart. Referer til fargebeskrivelsene for å vite hvilken farge du skal benytte mot hvilken bakgrunn for å ivareta kontrastkravet.
 
 #### Funksjonsfarger
+Funksjonsfargene rød, blå og grønn brukes for å gi informasjon om systemets tilstand. Rødt for feilmeldinger, grønt for suksess og blå for nøytral informasjon. Disse fargene skal kun benyttes til dette formålet og ikke i illustrasjoner eller grafikker (dette gjelder ikke for de støttefargene som også benyttes i samme funksjonsøyemed).
```

Comment:
Mer pirk:
For meg havner "Funksjonsøyemed" inn i samme kategori som "Hensynta". Er det mulig å forenkle/modernisere?